docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -158,159 +134,387 @@ $ tsh mfa add
-#   Logged in as:       codingllama
-#   Cluster:            example.com
-#   Roles:              access, editor, reviewer
-#   Logins:             codingllama
-#   Kubernetes:         enabled
-#   Valid until:        2021-10-04 23:32:29 -0700 PDT [valid for 12h0m0s]
-#   Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+$ tctl get cap > cap.yaml
 ```
 
-</TabItem>
+Ensure that `cap.yaml` contains the following content:
 
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+```yaml
+kind: cluster_auth_preference
+metadata:
+  name: cluster-auth-preference
+spec:
+  require_session_mfa: true
+version: v2
+```
 
 ```code
-$ tsh login --proxy=mytenant.teleport.sh
-# Enter password for Teleport user codingllama:
-# Tap any security key or enter a code from a OTP device:
-# > Profile URL:        https://mytenant.teleport.sh
-#   Logged in as:       codingllama
-#   Cluster:            mytenant.teleport.sh
-#   Roles:              access, editor, reviewer
-#   Logins:             codingllama
-#   Kubernetes:         enabled
-#   Valid until:        2021-10-04 23:32:29 -0700 PDT [valid for 12h0m0s]
-#   Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+$ tctl create -f cap.yaml
 ```
 
-</TabItem>
+Obtain your existing `cluster_auth_preference` resource:
 
-</Tabs>
+```code
+$ tctl get cap > cap.yaml
+```
 
-<Admonition type="note">
-  WebAuthn for logging in to Teleport is only required for [local users](
-  ../../reference/authentication.mdx#local-no-authentication-connector).
-  SSO users should configure multi-factor authentication in their SSO provider.
-</Admonition>
+Ensure that `cap.yaml` contains the following content:
 
-## U2F
+```yaml
+kind: cluster_auth_preference
+metadata:
+  name: cluster-auth-preference
+spec:
+  require_session_mfa: true
+version: v2
+```
 
-WebAuthn has replaced U2F in Teleport from previous versions. If you haven't configured U2F
-before, no further action is necessary—any U2F devices are automatically
-supported.
+### Step 2/2. Create the resource
 
-If you have an existing U2F configuration, but haven't explicitly configured
-WebAuthn yet, Teleport will automatically derive your WebAuthn configuration
-from your existing U2F configuration.
+```code
+$ tctl create -f cap.yaml
+```
 
-You may write the WebAuthn configuration yourself, but keep the U2F `app_id`
-field. Doing so ensures that any already-registered U2F devices won't need to be
-re-registered.
+### Per role
 
-For example, consider the U2F configuration below:
+To enforce MFA checks for a specific role, update the role to contain:
 
 ```yaml
-# snippet from /etc/teleport.yaml showing a U2F configuration:
-auth_service:
-  authentication:
-    type: local
-    second_factor: u2f
-    u2f:
-      app_id: https://example.com
-      facets:
-      - "https://example.com"
-      - "https://example.com:443"
-      device_attestation_cas:
-      - "/path/to/u2f_attestation_ca.pem"
```

Comment:
Some of this context looks important? Since this guide is replacing the webauthn guide, is it anywhere else?

Reply:
U2F support was deprecated many versions ago in favor of webauthn.

Reply:
Good point, I noticed we also removed  `attestation_allowed_cas:`  from the webauthn section, is this just to make the setup simpler? and hide this in a reference somewhere?

Reply:
Hi @benarent ! We still have this content for `attestation_allowed_cas` here:

https://goteleport.com/docs/access-controls/guides/webauthn/#step-13-enable-webauthn-support

Let me know if something is missing or if I should expound on this.

Thank you!